px4_ros2_cpp/test/integration/local_navigation.cpp

```diff
@@ -140,7 +140,8 @@ TEST_F(LocalPositionInterfaceTest, fuseEvVel) {
 
 TEST_F(LocalPositionInterfaceTest, fuseEvYaw) {
   auto measurement = std::make_unique<LocalPositionMeasurement>();
-  measurement->attitude_quaternion = Eigen::Quaternionf {1.0F, 0.0F, 0.0F, 0.0F};
+  measurement->attitude_quaternion =
```

Comment:
Can you add a comment that it expects the vehicle to point east?

Reply:
Added a comment, unfortunately I don't think this was the CI issue. It was only causing a warning but not causing the test to fail